docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -1,6 +1,12 @@
-# Use an IDE, dbx and Databricks Repos to develop a Kedro project
+# Use Databricks Asset Bundles to deploy a Kedro project
```

Comment:
I know we'll have to put a redirect but can we rename this file so that DAB or "asset bundles" is in the name?

Reply:
For better SEO

Reply:
I haven't done this before, I think we can only do the redirect after this is released? I can do the rename now.

Reply:
Yes, I'll take care of the redirect.